src/index.css

```diff
@@ -121,6 +121,16 @@ body {
   }
 }
 
+.badgeBlockName {
+  display: -webkit-box;
+  -webkit-line-clamp: 2;
```

Comment:
What about `Ff`? The `webkit` won't work with it

Reply:
We can use `tailwind` plugin for this: https://tailwindcss.com/docs/plugins#line-clamp

Reply:
Set plugin to project